src/main/java/com/github/visola/familymenu/controller/FamilyController.java

```diff
@@ -46,8 +46,15 @@ public Family getFamily(@PathVariable Integer id) {
         if (loadedFamily == null) {
             throw new ResourceNotFoundException("Family with ID " + id + " does not exist.");
         }
-        loadedFamily.setPassword("****");
-        return loadedFamily;
+        return hidePassword(loadedFamily);
+    }
+
+    private Family hidePassword(Family family) {
```

Comment:
I don't know if this is overkill but could you have a separate `FamilyCreationView` where you have an `id` field and a `name` field but then the `password` field is hardcoded to `****` so you can avoid the (slightly) confusing nature of a "new" Family being returned for the sake of returning information to the API caller?

Reply:
I like that. Much better.